scripts/validate-docs.sh

```
@@ -0,0 +1,162 @@
+    fi
+  done < <(grep -oP '\]\(\K[^)]+' "$mdfile" 2>/dev/null || true)
+done < <(find packages contributing -name "*.md" 2>/dev/null || true)
+
+if [ $BROKEN_LINKS -eq 0 ]; then
+  echo -e "${GREEN}✓ All internal links resolve${NC}"
+fi
+echo ""
+
+# Check 3: Code examples type-check
+echo "3️⃣  Checking TypeScript code examples..."
+CODE_ERRORS=0
```

Shellcheck (0.11.0)

[warning] CODE_ERRORS appears unused. Verify use (or export if used externally).

(SC2034)